### PR TITLE
fix: remove zjwt

### PR DESCRIPTION
**Pull request by kshamajain99**
Remove zjwt as for >v2.0 Argo CD is splitting JWT token if greater than standard size (4096). 

Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

**Review by alexmt — APPROVED** [review]
LGTM

File: server/server.go [hunk 1]
```diff
@@ -985,11 +977,11 @@ func getToken(md metadata.MD) string {
 	}
 
 	for _, t := range tokens {
```
Comment (alexmt) [inline server/server.go:979]: Can we remove tokens list and just return token as soon as it is extracted from grpc metadata:

line 973:
```golang
token, err := httputil.JoinCookies(common.AuthCookieName, request.Cookies())
if token != "" && err == nil {
	return token
}
```

I hope I'm not missing anything.

**Review by alexmt — CHANGES_REQUESTED** [review]

File: server/server.go (hunk 1 above)
Comment (kshamajain99) [reply server/server.go:979]: Yes, that is how I implemented earlier. But then what is the usecase of token list? https://github.com/argoproj/argo-cd/pull/5774/files#diff-78f42ba40d0f10b08c73b7e6bb8376f398e249c963cf549e89591d6b6826b9a4L972.

Comment (kshamajain99) [reply server/server.go:979]: To add one more point, https://github.com/argoproj/argo-cd/pull/5774/files#diff-78f42ba40d0f10b08c73b7e6bb8376f398e249c963cf549e89591d6b6826b9a4L988-L990 was removing invalid tokens (token with fragments != 3 when split by '.'). That is why I added it into to list and than ran through for loop to check token validity.

Comment (alexmt) [reply server/server.go]: Found PR that introduced `token` list: https://github.com/argoproj/argo-cd/pull/2262/files  Not 100% sure I correctly understand that code but I guess we are trying to get the first valid token (token has 3 fragments ). The Bearer token has a higher priority.


In this the logic you've implemented is correct. Sorry for confustion

**Review by alexmt — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**